scripts/apidocs-gen/config.yaml

```
@@ -2,11 +2,16 @@ processor:
   # RE2 regular expressions describing types that should be excluded from the generated documentation.
   ignoreTypes:
     - "List$"
+    # TODO: ignore the types not used in KIC like `KongService`, `KongRoute`.
   # RE2 regular expressions describing type fields that should be excluded from the generated documentation.
   ignoreFields:
     - "status$"
     - "TypeMeta$"
+  customMarkers:
+    - name: "apireference:kic:include"
+      target: type
 
 render:
   # Version of Kubernetes to use when generating links to Kubernetes API documentation.
-  kubernetesVersion: 1.25
+  # NOTE: The type of value here is `number`, so using versions ending with `0` like `1.30` will result in unexpected URL of kubernetes object meta in generated docs.
+  kubernetesVersion: 1.28
```

It will work as expected if we pass it as a string.
```suggestion
  kubernetesVersion: "1.30"
```